Alliance.lua

```diff
@@ -223,6 +228,17 @@ points[111] = { -- "ShattrathCity"
 	[56208180] = 12404, -- Scryer's Tier
 }
 
+if standingWithAldor < neutralStanding then
+	points[104][61002820] = nil -- ShadowmoonValley Altar of Sha'tar
+	points[111][28104900] = nil -- ShattrathCity Aldor Rise
+	
+end
+
+if standingWithScryers < neutralStanding then
```

Comment:
this doesn't working currently :\

Reply:
Annoyingly, it works when you reload...  The GetFactionInfo doesn't appear to give correct info on initial addon load.

Reply:
I'll put a delay on it when I merge it in. That should do the trick.

Reply:
Wait I found it. The function is GetFactionInfoByID, not GetFactionInfo.